history-sync.plugin.zsh

```diff
@@ -211,9 +211,13 @@ function history_sync_push() {
 
     # Encrypt
     if ! [[ "${#recipients[@]}" > 0 ]]; then
-        echo -n "Please enter GPG recipient name: "
-        read name
-        recipients+="$name"
+        if [[ -v ZSH_HISTORY_DEFAULT_RECIPIENT ]]; then
```

Comment:
`-v` is only available in Zsh 5.3+. Is there a POSIX-compliant way to achieve this?